DDSim/DDSim/Helper/ParticleHandler.py

```diff
@@ -13,6 +13,18 @@ def __init__( self ):
     self._printEndTracking = False
     self._printStartTracking = False
     self._minDistToParentVertex = 2.2e-14*mm
+    self._enableDetailedHitsAndParticleInfo = False
+    self._enableDetailedHitsAndParticleInfo_HELP = "print out lots of info about MCHistory"
```

Comment:
if you use property, you don't need the "_HELP" variable, or vice versa., and in case of "_HELP" the original variable must not have a leading underscore, because variables with leading underscore are ignored and the property is used instead to add the argument parsing.
i.e.
```
self.enableDetailedHitsAndParticleInfo = False
self._enableDetailedHitsAndParticleInfo_HELP = "print out lots of info about MCHistory"
```